### DISCUSSION TITLE
How to configure the input style in the template?

### DISCUSSION THREAD

**Discussion by FeiPengDev** [vueform/builder#83] [Questions] [answered]
![image](https://github.com/user-attachments/assets/f8ef5747-8767-45f4-95d0-db967e86a803)

Excuse me, please tell me how to configure the input style in the template. Is the style of putting label inside input realized through configuration or custom element?

**Comment by adamberecz** [comment, marked as answer]
Hi @FeiPengDev—apologies for the late reply, I missed this question somehow.

That's called a **floating label** and can be configured via the following [CSS vars](https://vueform.com/docs/styles-and-layout#css-vars):
- `--vf-color-floating`
- `--vf-color-floating-focus`
- `--vf-color-floating-success`
- `--vf-color-floating-danger`

These CSS vars can also be set via the builder's Theme panel under Input section. [Here](https://builder.vueform.com/docs/rendering#theme) you can find the docs on how to use the Theme export of the builder to convert them to CSS variables.

If you ever experience delay in my reply here, please send me an email to adam@vueform.com.
